src/Microsoft.AspNet.Http.Abstractions/project.json

```diff
@@ -2,6 +2,7 @@
   "version": "1.0.0-*",
   "description": "ASP.NET 5 HTTP object model. HttpContext and family.",
   "dependencies": {
+    "Microsoft.AspNet.Http.Features": "1.0.0-*",
```

Comment:
@Tratcher : I did not check this in actually...i wanted to combine Session related changes with the checkin...